### ISSUE TITLE
v3.3.8 and v3.3.9 fail typescript tsc

### ISSUE THREAD

**Issue by CharlesSwiftConnect** [googlemaps/google-maps-services-js#753]
@googlemaps/google-maps-services-js@3.3.8 and 3.3.9 fail Typescript tsc. @googlemaps/google-maps-services-js@3.3.7 passes without issue.

#### Steps to reproduce

  1. yarn add @googlemaps/google-maps-services-js@3.3.8
      alternatively:  yarn add @googlemaps/google-maps-services-js@3.3.9
  2. yarn tsc

#### Stack trace
```
yarn run v1.22.17
$ /xxx/node_modules/.bin/tsc
src/modules/google/commands/get-geolocalize-locality-cmd.ts:19:31 - error TS2322: Type 'import("/xxx/node_modules/axios/index").AxiosInstance' is not assignable to type 'import("/xxx/node_modules/@googlemaps/google-maps-services-js/node_modules/axios/index").AxiosInstance'.
  The types of 'defaults.transformRequest' are incompatible between these types.
    Type 'import("/xxx/node_modules/axios/index").AxiosRequestTransformer | import("/xxx/node_modules/axios/index").AxiosRequestTransformer[] | undefined' is not assignable to type 'import("/xxx/node_modules/@googlemaps/google-maps-services-js/node_modules/axios/index").AxiosRequestTransformer | import("/xxx/node_modules/@googlemaps/google-maps-services-js/node_modules/axios/index").AxiosRequestTransformer[] | undefined'.
      Type 'AxiosRequestTransformer' is not assignable to type 'AxiosRequestTransformer | AxiosRequestTransformer[] | undefined'.
        Type 'import("/xxx/node_modules/axios/index").AxiosRequestTransformer' is not assignable to type 'import("/xxx/node_modules/@googlemaps/google-maps-services-js/node_modules/axios/index").AxiosRequestTransformer'.
          Types of parameters 'headers' and 'headers' are incompatible.
            Type 'import("/xxx/node_modules/@googlemaps/google-maps-services-js/node_modules/axios/index").AxiosRequestHeaders | undefined' is not assignable to type 'import("/xxx/node_modules/axios/index").AxiosRequestHeaders | undefined'.
              Type 'import("/xxx/node_modules/@googlemaps/google-maps-services-js/node_modules/axios/index").AxiosRequestHeaders' is not assignable to type 'import("/xxx/node_modules/axios/index").AxiosRequestHeaders'.
                'string' index signatures are incompatible.
                  Type 'string | number | boolean' is not assignable to type 'string'.
                    Type 'number' is not assignable to type 'string'.

19   const client = new Client({ axiosInstance })
                                 ~~~~~~~~~~~~~


Found 1 error.

error Command failed with exit code 2.
```

**Comment by jpoehnelt** [comment]
@CharlesSwiftConnect Thank you for opening this issue. 🙏
Please check out these other resources that might be applicable:

- [Support Console](https://cloud.google.com/support/) - If you have a support contract.
- [Discord](https://discord.gg/9UpV862xZ9) - chat with other developers
- [StackOverflow](http://stackoverflow.com/questions/tagged/google-maps) - use the `google-maps` tag
- [Google Maps Platform issue trackers](https://developers.google.com/maps/support/#issue_tracker)


_This is an automated message, feel free to ignore._

**Comment by CharlesSwiftConnect** [comment]
Fixed in 3.3.10
